tools/SourceKit/lib/SwiftLang/SwiftCompletion.cpp

```diff
@@ -152,7 +152,9 @@ static bool swiftCodeCompleteImpl(SwiftLangSupport &Lang,
   if (Failed) {
     return false;
   }
-  if (!Invocation.getFrontendOptions().InputsAndOutputs.hasInputs()) {
+
+  auto &InputsAndOutputs = Invocation.getFrontendOptions().InputsAndOutputs;
```

Comment:
My own style would not favor introducing the local variable here. The local variable does have the benefit of making the references to it more concise. But it has the cost of implicitly suggesting to the reader that this  entity is used and modified for the whole rest of this function. The function is long enough that, as a reader, I have to spend a bit of time checking for uses of this variable for the rest of the function. I guess another way to explain my feeling is that the declaration of the variable does not carry information about why it is being declared. So, for me the tradeoff would lean towards not declaring the variable.